### PR TITLE
Announce the contributor summit schedule

### PR DESCRIPTION
**Pull request by castrojo**
This is to announce the Amsterdam contributor summit. Acked/reviewed by the summit staff.

### CODE REVIEW

File: content/en/blog/_posts/2020-02-18-Contributor-Summit-Amsterdam-Schedule-Announced.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+*   Community
+*   Contributor Improvement
+*   Sustainability
+*   In-depth Technical 
+
+On top of the presentations, there will be a dedicated Docs Sprint as well as the New Contributor Workshop 101 and 201 Sessions. All told, we will have five separate rooms of content throughout the day on Monday. Please **[see the full schedule](https://kcseu2020.sched.com/)** to see what sessions you’d be interested in. We hope between the content provided and the inevitable hallway track, everyone has a fun and enriching experience. 
+
+Speaking of fun, the social Sunday night should be a blast! We’re hosting this summit’s social close to the conference center, at [ZuidPool](https://www.zuid-pool.nl/en/). There will be games, bingo, and unconference sign-up throughout the evening and should be a relaxed way to kick off the week.
```
Comment (sftim) [inline content/en/blog/_posts/2020-02-18-Contributor-Summit-Amsterdam-Schedule-Announced.md]: nit (bearing in mind “Acked/reviewed by the summit staff”):
```suggestion
Speaking of fun, the social Sunday night should be a blast! We’re hosting this summit’s social close to the conference center, at [ZuidPool](https://www.zuid-pool.nl/en/). There will be games, bingo, and unconference sign-up throughout the evening. It should be a relaxed way to kick off the week.
```

File: content/en/blog/_posts/2020-02-18-Contributor-Summit-Amsterdam-Schedule-Announced.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+*   In-depth Technical 
+
+On top of the presentations, there will be a dedicated Docs Sprint as well as the New Contributor Workshop 101 and 201 Sessions. All told, we will have five separate rooms of content throughout the day on Monday. Please **[see the full schedule](https://kcseu2020.sched.com/)** to see what sessions you’d be interested in. We hope between the content provided and the inevitable hallway track, everyone has a fun and enriching experience. 
+
+Speaking of fun, the social Sunday night should be a blast! We’re hosting this summit’s social close to the conference center, at [ZuidPool](https://www.zuid-pool.nl/en/). There will be games, bingo, and unconference sign-up throughout the evening and should be a relaxed way to kick off the week. 
+
+[Registration is open](https://events.linuxfoundation.org/kubernetes-contributor-summit-europe/)! Space is limited so it’s always a good idea to register early. 
+
+If you have any questions, reach out to the [Amsterdam Team](https://github.com/kubernetes/community/tree/master/events/2020/03-contributor-summit#team) on slack in the [#contributor-summit](https://kubernetes.slack.com/archives/C7J893413) channel.
```
Comment (sftim) [inline content/en/blog/_posts/2020-02-18-Contributor-Summit-Amsterdam-Schedule-Announced.md]: nit (bearing in mind “Acked/reviewed by the summit staff”):
```suggestion
If you have any questions, reach out to the [Amsterdam Team](https://github.com/kubernetes/community/tree/master/events/2020/03-contributor-summit#team) on Slack in the [#contributor-summit](https://kubernetes.slack.com/archives/C7J893413) channel.
```

File: content/en/blog/_posts/2020-02-18-Contributor-Summit-Amsterdam-Schedule-Announced.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+**Authors:** Jeffrey Sica (Red Hat), Amanda Katona (VMware) 
+
+tl;dr [Registration is open](https://events.linuxfoundation.org/kubernetes-contributor-summit-europe/) and the [schedule is live](https://kcseu2020.sched.com/) so register now and we’ll see you in Amsterdam!
+
+## Kubernetes Contributor Summit 
+
+**Sunday, March 29, 2020**
+- Evening Contributor Celebration:
+[ZuidPool](https://www.zuid-pool.nl/en/)
+- Address: Europaplein 22, 1078 GZ Amsterdam[, Netherlands](https://www.google.com/search?q=kubecon+amsterdam+2020&oq=kubecon+amste&aqs=chrome.0.35i39j69i57j0l4j69i61l2.3957j1j4&sourceid=chrome&ie=UTF-8&ibp=htl;events&rciv=evn&sa=X&ved=2ahUKEwiZoLvQ0dvnAhVST6wKHScBBZ8Q5bwDMAB6BAgSEAE#)
```
Comment (sftim) [inline content/en/blog/_posts/2020-02-18-Contributor-Summit-Amsterdam-Schedule-Announced.md]: Is this clearer?
```suggestion
- Address: [Europaplein 22, 1078 GZ Amsterdam, Netherlands](https://www.google.com/search?q=KubeCon+Amsterdam+2020&ie=UTF-8&ibp=htl;events&rciv=evn&sa=X&ved=2ahUKEwiZoLvQ0dvnAhVST6wKHScBBZ8Q5bwDMAB6BAgSEAE#)
```

**Comment by castrojo** [comment]
/hold

Forgot the picture. :dancer:

**Comment by sftim** [comment]
https://deploy-preview-19174--kubernetes-io-master-staging.netlify.com/blog/2020/02/18/contributor-summit-amsterdam-schedule-announced/ /lgtm (informally)

**Pull request merged**